demos/multi_camera_multi_target_tracking_demo/python/utils/network_wrappers.py

```diff
@@ -133,15 +133,29 @@ def __init__(self, ie, model_path, trg_classes, conf=.6,
         self.confidence = conf
         self.net = load_ie_model(ie, model_path, device, None, ext_path, num_reqs=self.max_reqs)
 
-        required_input_keys = [{'im_info', 'im_data'}, {'im_data', 'im_info'}]
-        current_input_keys = self.net.input_info.keys()
-        assert current_input_keys in required_input_keys
-        required_output_keys = {'boxes', 'scores', 'classes', 'raw_masks'}
-        assert required_output_keys.issubset(self.net.net.outputs)
+        required_input_keys = [{'image'}]
+        required_output_keys = {'boxes', 'labels', 'masks'}
 
-        self.n, self.c, self.h, self.w = self.net.input_info['im_data'].input_data.shape
+        required_input_keys_segmentoly = [{'im_info', 'im_data'}, {'im_data', 'im_info'}]
```

Comment:
These two sets are equal. The same is for `check_segmentoly_type()` and whiteboard_inpainting_demo.

Reply:
Fixed.